pyairtable/api/abstract.py

```diff
@@ -64,7 +66,7 @@ def _options_to_params(self, **options):
     def _chunk(self, iterable, chunk_size):
         """Break iterable into chunks"""
         for i in range(0, len(iterable), chunk_size):
-            yield iterable[i : i + chunk_size]
+            yield iterable[i: i + chunk_size]
```

Comment:
as a nitpick you modified some lines with just whitespace changes, sometimes even ones against Python's style guidelines (there are few more places with unnecessary double spaces, no whitespace at all, etc.)

Reply:
I made my changes [PEP8](https://www.python.org/dev/peps/pep-0008/) compliant

Reply:
i actually don't think so!

<img width="816" alt="Screenshot 2022-03-04 at 18 28 43" src="https://user-images.githubusercontent.com/5056119/156821002-bd0e00da-dc23-442c-9ac9-1bb36930427a.png">

either way – doesn't matter, you should run this to autoformat the entire project:

https://github.com/gtalarico/pyairtable/blob/bb13d4c06e3917682c1900afbb14f88ba65adc5a/Makefile#L41-L42

I'm surprised that CI didn't include a check that the files are formatted! 🤔

Reply:
dayum, my IDE was lying to me there, I guess i should not try the built in formatter :(

Reply:
It is supposed to run flake8 and black in CI but I think I forgot to configure it ! 🙈 
The CI workflow is even called `test, lint, deploy` :)
https://github.com/gtalarico/pyairtable/blob/main/.github/workflows/test_lint_deploy.yml
https://github.com/gtalarico/pyairtable/blob/main/tox.ini#L18